plugins/module_utils/aci.py

```diff
@@ -1429,10 +1433,25 @@ def get_existing(self):
         that this method can be used to supply the existing configuration when using the get_diff method. The response, status,
         and existing configuration will be added to the self.result dictionary.
         """
-        uri = self.url + self.filter_string
+        if self.suppress_previous:
+            self.existing = []
+            return
 
+        uri = self.url + self.filter_string
         self.api_call("GET", uri, data=None, return_response=False)
 
+    def __get_existing_validation(self, changed):
```

Comment:
It would be helpful to have a short description of what this function does like we already did for all existing functions under the ACIModule class. for example, like the short description given in get_existing function.

Reply:
Added docstring 👍